### PR TITLE
Writing & Guide Update to evc.md

### PR DESCRIPTION
**Pull request by NecroNovaVGitHub**
Part 230409 of me adding writing to the site

### CODE REVIEW

File: src/content/guide/evc.md [hunk 1]
```diff
@@ -24,7 +50,12 @@ ___
 3. Install the Everybody Votes Channel and Region Select channel using the provided WAD installer or the one of your choice.
 5. Return to the Wii Menu and open the Everybody Votes Channel.
 
-#### With any luck, you'll be up and running!
+#### With any luck, you'll be up and running! 
+
+* Here is an Infographic for people with reading issues, such as Dyslexia.
+
+![EVC Transfer Infographic](https://media.discordapp.net/attachments/997708025169203212/1155280197499752528/Switch_Providers_Inforgraphic.png)
```
Comment (hytracer) [inline src/content/guide/evc.md:57]: Please do not use discord links. Add your files in the public directory

File: src/content/guide/evc.md [hunk 2]
```diff
@@ -24,7 +50,12 @@ ___
...
+</br>
```
Comment (hytracer) [inline src/content/guide/evc.md:58]:
```suggestion

```
Don't use new lines

File: src/content/guide/evc.md [hunk 3]
```diff
@@ -6,16 +6,42 @@ pubDate: "2023-08-26"
...
+5. Highlight the WADs with `Everybody Votes Channel (WiiLink)` and `Region Select` as its name and press + to select it. When it is selected, press A twice to install the WADs.
+> Both WAD names might also have `[USA]`, `[PAL]`, or `[Japan]` depending on what region of Console youre installing the Channel on. Please install the correct region version of the Channel on your Console.
+#### If you happen to get `Error -1022` when installing `Everybody Votes Channel (WiiLink)` or `Region Select`, uninstall the channels by selecting the WAD with `-` and pressing `A`. Then, re-install the WAD.
 
-## We're moving! - How to Transfer your Data
+6. After it is successfully installed, press the HOME Button to exit back to the Homebrew Channel. Press the HOME Button again and select `"Exit to System Menu"`.
 
+### Congrats! You've installed the Everybody Votes Channel on your Console! Enjoy the Channel to your hearts content once again!
+
+> If you would like to migrate your poll data from RiiConnect24 over to WiiLink, please use the EVC Transfer Tool by following the instructions below.
```
Comment (hytracer) [inline src/content/guide/evc.md:41]:
```suggestion
> If you wish to migrate your poll data from RiiConnect24 over to WiiLink, please use the EVC Transfer Tool by following the instructions below.
```

**Review by hytracer — CHANGES_REQUESTED** [review]

**Comment by NecroNovaVGitHub** [comment]
All of these suggestions will absolutely be fixed in the planned major writing update. Thank you for noticing these issues.

**Pull request closed without merging**